packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -552,6 +564,14 @@ function BackgroundSizeControls( {
 				value={ backgroundPositionToCoords( positionValue ) }
 				onChange={ updateBackgroundPosition }
 			/>
+			<ToggleControl
```

Comment:
I've put no thought put into this location or the actual control. Could you tell? 🤣 

It does affect the height of the popover, so maybe there's some optimization required.

Elsewhere I've tried using a select dropdown for the size options, and having it site to the side, e.g.,

<img width="405" alt="Screenshot 2024-07-05 at 11 47 45 AM" src="https://github.com/WordPress/gutenberg/assets/6458278/32cb0d09-f9dd-4148-9fc0-19351bb014cf">

The problem is that translations can and do push out the copy.

I wonder if an accordion or something similar might help to use the space more evenly?

The help text for this toggle could be removed as well.